apis/openstack-heat/src/main/java/org/jclouds/openstack/heat/v1/domain/Attributes.java

```diff
@@ -0,0 +1,184 @@
+/**
+ * 
+ */
+package org.jclouds.openstack.heat.v1.domain;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.beans.ConstructorProperties;
+
+import javax.inject.Named;
+
+import org.jclouds.openstack.v2_0.domain.Link;
+import org.jclouds.openstack.v2_0.services.Image;
+
+import com.google.common.base.Objects;
+import com.google.common.base.Objects.ToStringHelper;
+
+/**
+ * @author deviyada
+ *
+ */
+public class Attributes {
+
+	@Named("id")
```

Comment:
you don't need the @Named if the field have the same name as the params (please take a look on stack)